hat/urls.py

```diff
@@ -20,123 +20,130 @@
 admin.site.index_title = "Administration de Iaso"
 
 
-if settings.DISABLE_PASSWORD_LOGINS:
-    login_template = "iaso/disabled_password_login.html"
-    urlpatterns = [
-        path("admin/login/", TemplateView.as_view(template_name=login_template), name="admin-login"),
-        path("login/", TemplateView.as_view(template_name=login_template), name="login"),
+if settings.MAINTENANCE_MODE:
+    maintenance_template = "iaso/maintenance.html"
+    urlpatterns = [path("", TemplateView.as_view(template_name=maintenance_template), name="maintenance")]
```

Comment:
Should we also use this template as the default one for all 404s?

Reply:
all 404 in maintenance mode, you mean? I think that's a good idea!

Reply:
Yup, to let people know that there is an ongoing maintenance instead of just the default 404 template.

We should be able to override `handler404` or something like that.